File: Examples/Example2/Example2/ViewController.m
//
//  ViewController.m
//  Example2
//
//  Created by qianzhao on 2018/5/11.
//  Copyright © 2018年 qianzhao. All rights reserved.
//

#import "ViewController.h"
#import "RootViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = @"RootVC";
    // Do any additional setup after loading the view, typically from a nib.
    
    RootViewController *vc = [RootViewController new];
    vc.title = @"TX Beacon Test";
    
    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.8f * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
        [self.navigationController pushViewController:vc animated:YES];
    });
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Examples/Example1/Example/ViewController.swift
//
//  ViewController.swift
//  Example
//
//  Created by qianzhao on 2018/5/10.
//  Copyright © 2018年 qianzhao. All rights reserved.
//

import UIKit
import SnapKit
import RxSwift
import RxCocoa
import QBToast
import QZCommon

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        initUI()
        configureEvent()
    }
    
    func initUI() -> Void {
        view.addSubview(button1)
        button1.snp.makeConstraints { (maker) in
            maker.width.equalTo(200)
            maker.height.equalTo(60)
            maker.center.equalToSuperview()
        }
    }
    
    func configureEvent() -> Void {
        button1.rx.tap.subscribe(onNext: { () in
            print("test")
            QBToaster.showText("show text", pos: .center)
            let a = TestClass1()
            print("a = \(a)")
        }).disposed(by: disnoseBag)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    let button1: UIButton = {
        let but = UIButton()
        but.backgroundColor = .red;
        but.setTitle("test", for: .normal)
        but.setTitle("testHL", for: .highlighted)
        return but
    }()
    let disnoseBag = DisposeBag()
}

